### PR TITLE
Fixes from codacy

### PR DESCRIPTION
**Pull request by EmmanuelMess**
#### Issue tracker   
Fixes will automatically close the related issue

Fixes #
-or-   
Addresses #

#### Release  
Addresses release/
  
#### Test cases
- [ ] Covered
  
#### Manual testing
- [ ] Done  
  
If yes,  
- Device:
- OS:

#### Build tasks success  
Successfully running following tasks on local 
- [x] `./gradlew assembledebug`
- [x] `./gradlew spotlessCheck`

#### Related PR  
Related to PR #

#### Additional Info

### CODE REVIEW

File: app/src/main/java/com/amaze/filemanager/ui/fragments/MainFragment.java [hunk 1]
```diff
@@ -476,8 +478,9 @@ public boolean onPrepareActionMode(ActionMode mode, Menu menu) {
             return true;
           }
 
-          if (getMainActivity().mReturnIntent)
-            if (Build.VERSION.SDK_INT >= 16) showOption(R.id.openmulti, menu);
+          if (getMainActivity().mReturnIntent && Build.VERSION.SDK_INT >= 16) {
```
Comment (TranceLove) [inline app/src/main/java/com/amaze/filemanager/ui/fragments/MainFragment.java:481]: Perhaps `Build.VERSION.SDK_INT >= JELLY_BEAN` is better?

File: app/src/main/java/com/amaze/filemanager/ui/activities/MainActivity.java [hunk 2]
```diff
@@ -168,6 +169,8 @@
         LoaderManager.LoaderCallbacks<Cursor>,
         FolderChooserDialog.FolderCallback {
 
+  private static final String TAG = "MainActivity";
```
Comment (TranceLove) [inline app/src/main/java/com/amaze/filemanager/ui/activities/MainActivity.java:172]: Perhaps `MainActivity.class.getSimpleName()` is better?

**Review by TranceLove — DISMISSED** [review]

File: app/src/main/java/com/amaze/filemanager/TagsHelper.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.amaze.filemanager;
+
+public class TagsHelper {
+  public static String getTag(Class<?> clazz) {
+    return clazz.getSimpleName().substring(0, 23);
```
Comment (TranceLove) [inline app/src/main/java/com/amaze/filemanager/TagsHelper.java:25]: Why you will need `substring()` when `Class.getSimpleName()` would only return the class' own name without package?

**Review by TranceLove — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/amaze/filemanager/TagsHelper.java (hunk 3 above)
Comment (EmmanuelMess) [reply app/src/main/java/com/amaze/filemanager/TagsHelper.java:25]: Someone might have a class that is exceedingly long, especially in Kotlin or for nested classes, where the TAG will end up with >23 chars, witch will directly crash in production.

Comment (TranceLove) [reply app/src/main/java/com/amaze/filemanager/TagsHelper.java:25]: But what if the class name is shorter than 23 chars? This should be the reason why the tests ended up with `IndexOutOfBoundException`s

**Review by TranceLove — APPROVED** [review]

**Review by VishalNehra — APPROVED** [review]

**Pull request merged**